docs/faq.md

```diff
@@ -32,8 +32,8 @@
 
     Since the wire in the jig is supposed to touch the pads inside the Switch's joycon rail, it is important to use wire that is thin, not rigid and bent/not pointy. 
     Paperclips make for potentially dangerous jigs, as they are made out of a hard material, are rigid and pointy and can easily scratch off the pads inside the Switch. 
-    A good jig uses 32Gauge(0.2mm diameter) wire and is bent in a way that the end of the wire does not scratch the pads. <a href="https://switchjigs.com/" target="_blank">Switch Jigs</a> sells jigs we consider "good". 
-    You can also download and 3d-print your own jig and use the pictures on the website to guide you on how to bend the wire correctly.
+    A good jig uses 32Gauge(0.2mm diameter) wire and is bent in a way that the end of the wire does not scratch the pads.
+    You can download and 3d-print your own jig and use the pictures on the website to guide you on how to bend the wire correctly.
```

Comment:
this makes no sense now; please add a link to a good stl file or something for 3d printing. we used to use switchjigs for that, but since we're not recommending them, we do need a replacement for this.